### ISSUE TITLE
Click version miss-match

### ISSUE THREAD

**Issue by sobolevn** [seddonym/layer_linter#90]
* Layer Linter version: `>=0.12.0,<0.13.0`
* Python version: `3.6.6`
* Operating System: macos

### Description

I am trying to use `click@7` with `layer-linter`.
And the version resolution fails. 

### What I Did

I am using `poetry` to manage my dependencies.
To reproduce my issue:

1. Install the latest version of `poetry`
2. Create a new project with `poetry init`
3. Run `poetry add click` and specify a version `7`
3. Run `poetry add --dev layer-lint`

You will see something like this:

```
» poetry add click
Using version ^7.0 for click

Updating dependencies
Resolving dependencies... (0.1s)

[SolverProblemError]
Because no versions of layer-linter match >0.12.0,<0.13.0
 and layer-linter (0.12.0) depends on click (>=6.7,<7.0), layer-linter (>=0.12.0,<
0.13.0) requires click (>=6.7,<7.0).
So, because wemake-python-styleguide depends on both click (^7.0) and layer-linter
 (^0.12.0), version solving failed.
```

**Comment by seddonym** [comment]
Thanks for the bug report, I will look into this when I have a chance.

**Comment by seddonym** [comment]
Fixed in version 0.12.1.
